### DISCUSSION TITLE
List for user login

### DISCUSSION THREAD

**Discussion by DavevonH** [neuefische/questions#757] [Web] [answered]
We want to have a list of characters where the user of the app can select one of them and use it as his character. We are not sure how to pass the ID of the user and how to use it to fetch the information of this explicit character. Can someone help us with this problem?

Here is our [PR](https://github.com/Rick1Marques/star-wars-bazar-app/pull/34) for this problem.

![Bildschirmfoto 2023-10-10 um 14 55 43](https://github.com/neuefische/questions/assets/140090722/e8dc518f-9634-4e3c-b980-857ac60a9ba3)

**Comment by JessicaLoers** [comment, marked as answer]
**Hey @DavevonH**  ✌️
It would be fabulous if you could add the discussed solution here and mark this question as answered.
Frogs for ever  🐸
